indexer/README.md

```diff
@@ -2,225 +2,119 @@
-        "format"
-      ]
-    }
-  ],
-  "cell.json":[
-    "type",
-    "ontology",
-    "id"
-  ]
- }
-```
-***NOTE***: The config should be rooted under a version of the metadata being received.
-
-In Elasticsearch, the fields for the File Indexer will be
-```
-assay,json|rna|primer
-assay,json|single_cell|method
-assay,json|sra_experiment
-assay,json|files|format
-cell,json|type
-cell,json|ontology
-cell,json|id
-```
-Notice the commas(,) where there were previously periods(.). Also, the pipe (|) is used as the separator between levels in the config.
-
-#### Adding Mappings
-Given a config:
-```
-{
-  "cell.json":[
-    "type",
-    "ontology",
-    "id"
-  ]
- }
-```
-
-### Environment Variables
-In order to add environmental variables to Chalice, the variables must be added to three locations.
-Do not add protocols to any of the Endpoints. Make sure the ES_ENDPOINT does not have any trailing slashes.
-
-1) Edit Chalice  
-open `.chalice/config.json`  
-Replace the current file with the following, making sure to replace the <> with your information.  
-```
-{
-  "version": "2.0",
-  "app_name": "<your-indexer-lambda-application-name>",
-  "stages": {
-    "dev": {
-      "api_gateway_stage": "api",
-      "manage_iam_role":false,
-      "iam_role_arn":"arn:aws:iam::<AWS-account-ID>:role/<your-indexer-lambda-application-name>-dev",
-      "environment_variables": {
-         "ES_ENDPOINT":"<your elasticsearch endpoint>",
-         "BLUE_BOX_ENDPOINT":"<your blue box>",
-         "ES_INDEX":"<elasticsearch index to use>",
-         "INDEXER_NAME":"<your-indexer-lambda-application-name>",
-         "HOME":"/tmp"
+      "version": "2.0",
+      "app_name": "<your-indexer-lambda-application-name>",
+      "stages": {
+        "dev": {
+          "api_gateway_stage": "api",
+          "manage_iam_role":false,
+          "iam_role_arn":"arn:aws:iam::<AWS-account-ID>:role/<your-indexer-lambda-application-name>-dev",
+          "environment_variables": {
+             "ES_ENDPOINT":"<your elasticsearch endpoint>",
+             "BLUE_BOX_ENDPOINT":"<your blue box>",
+             "ES_INDEX":"<elasticsearch index to use>",
+             "INDEXER_NAME":"<your-indexer-lambda-application-name>",
+             "HOME":"/tmp"
+          }
+        }
       }
     }
-  }
-}
-```   
-
-2) Edit .profile
-open `~/.profile` and add the following items.
-
-```
-export ES_ENDPOINT=<your elasticsearch endpoint>
-export BLUE_BOX_ENDPOINT=<your blue box>
-export ES_INDEX=<elasticsearch index to use>
-export INDEXER_NAME=<your-indexer-lambda-application-name>
-export HOME=/tmp
-```
-
-run `. ~/.profile` to load the variables
-
-3) Edit Lambda
-
-Go to the AWS console, and then to your Lambda function and add the following environment variables:
-
-```
-ES_ENDPOINT  -->   <your elasticsearch endpoint>
-BLUE_BOX_ENDPOINT   -->   <your blue box>
-ES_INDEX  -->  <elasticsearch index to use>
-INDEXER_NAME  -->  <your-indexer-lambda-application-name>
-HOME --> /tmp
-```
-
```

Comment:
Need to add a new section here on reindexing using `find_golden_tickets.py` (or `reindex.py` as it is now called on the `develop` branch.